File: 709-to-lower-case/709-to-lower-case.c
char * toLowerCase(char * s){
    int i = 0;
    
    while(s[i])
    {
        if(s[i] >= 65 && s[i] <= 90)
            s[i] = s[i] + 32;
        ++i;
    }
    return s;
}

File: 709-to-lower-case/709-to-lower-case.java
class Solution {
    public String toLowerCase(String s) {
        int i = 0;
        char[] str = new char[s.length()];
        
        while (i < s.length()){
            if (s.charAt(i) >= 'A' && s.charAt(i) <= 'Z')
                str[i] = (char)((int)s.charAt(i) + 32);
            else
                str[i] = s.charAt(i);
            i++;
        }
        String newStr = String.valueOf(str);
        return newStr;
    }
}